python/xorbits/lightgbm/tests/test_classifier.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2022-2023 XProbe Inc.
+# derived from copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import xorbits.numpy as mt
+import xorbits.pandas as md
+
+try:
+    import lightgbm
+except ImportError:
+    lightgbm = None
+from ... import lightgbm as lgb
+
+n_rows = 1000
+n_columns = 10
+chunk_size = 200
+rs = mt.random.RandomState(0)
```

Comment:
Use `xorbits.numpy as np`, not mars tensor as inputs.

Reply:
I imported `xorbits.numpy` as `mt`.